ansible/roles/sidekiq/templates/sidekiq-monit-rc

```diff
@@ -0,0 +1,7 @@
+{% for queue in job_queues %}
+check process sidekiq_{{queue}}
+  with pidfile /u/apps/{{project_name}}/shared/tmp/pids/sidekiq.{{queue}}.pid
+  start program = "/bin/su - deploy -c 'cd /u/apps/{{project_name}}/current && bundle exec sidekiq --queue {{queue}} --pidfile /u/apps/{{project_name}}/shared/tmp/pids/sidekiq.{{queue}}.pid --environment {{rails_env}} --logfile /u/apps/{{project_name}}/shared/log/sidekiq.{{queue}}.log --daemon'" with timeout 30 seconds
+  stop program = "/bin/su - deploy -c 'kill -s TERM `cat /u/apps/{{project_name}}/shared/tmp/pids/sidekiq.{{queue}}.pid`'" with timeout 30 seconds
```

Comment:
I also don't know about the timeouts, can we leave those off until it's a problem? also does this really just use `kill` to shut it down? is that the best way?